go/v1/resource_descriptor_test.go

```diff
@@ -68,12 +70,22 @@ func TestBadResourceDescriptor(t *testing.T) {
 	assert.ErrorIs(t, err, ErrRDRequiredField, "created malformed ResourceDescriptor")
 }
 
-func TestBadResourceDescriptorDigest(t *testing.T) {
+func TestBadResourceDescriptorDigestEncoding(t *testing.T) {
```

Comment:
Perhaps I missed them, but are there any tests for 'good' digest encodings?

Reply:
Ah, good catch. We should add an explicit test for a valid supported digest (encoding and length).